### PR TITLE
Feature/header and keyboard accessibility

### PR DESCRIPTION
**Pull request by ForeverTangent**
OK First PR, part deus - I think I figured out how to submit without my project files.
But the extra time, helped me figure out a more VoiceOver way to enhance custom controls.
Also Remembered NSLocalized Strings this time.

### CODE REVIEW

File: AudioKitSynthOne/Keyboard/KeyboardSettingsViewController.swift [hunk 1]
```diff
@@ -75,3 +75,22 @@ class KeyboardSettingsViewController: UIViewController {
...
+		keyboardModeSegment.subviews[2].accessibilityLabel = NSLocalizedString("Dark", comment: "Dark")
+		
+		octaveRangeSegment.subviews[0].accessibilityLabel = NSLocalizedString("1", comment: "1")
+		octaveRangeSegment.subviews[1].accessibilityLabel = NSLocalizedString("2", comment: "2")
+		octaveRangeSegment.subviews[2].accessibilityLabel = NSLocalizedString("3", comment: "3")
+		
+		labelModeSegment.subviews[0].accessibilityLabel = NSLocalizedString("None", comment: "None")
+		labelModeSegment.subviews[1].accessibilityLabel = NSLocalizedString("C", comment: "C")
+		labelModeSegment.subviews[2].accessibilityLabel = NSLocalizedString("All", comment: "All")
+	}
```
Comment (megastep) [inline AudioKitSynthOne/Keyboard/KeyboardSettingsViewController.swift]: Could use a bit more context in the comments here, for the translators' sake, but I'm nit-picking :)
Also not entirely sure it was necessary to localize just numbers, but doesn't really matter I guess...

Comment (analogcode) [reply AudioKitSynthOne/Keyboard/KeyboardSettingsViewController.swift]: That's to turn on/off the label notes on the keyboard. Is that useful for accessibility? Adding additional visual labeling?

Comment (megastep) [reply AudioKitSynthOne/Keyboard/KeyboardSettingsViewController.swift]: Mostly these values are used to be read by the VoiceOver when hitting the buttons. The whole interface must be readable.

Comment (ForeverTangent) [reply AudioKitSynthOne/Keyboard/KeyboardSettingsViewController.swift]: Megastep, is correct.  There are exceptions, but in general you try and get everything pertinent for VoiceOver.   It is not just those which have complete vision loss which use VO, but people with low vision or physical/cognitive impairments need it as well. Sort of as a way to double check the control they are trying to use is the correct one.

Then you have the weirdos like me, who can see fine but use it so I don't have to look at the phone.

**Pull request merged**